jar-infer/jar-infer-lib/src/test/java/com/uber/nullaway/jarinfer/AnnotationComparator.java

```diff
@@ -0,0 +1,141 @@
+package com.uber.nullaway.jarinfer;
+
+import com.google.common.base.Preconditions;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Enumeration;
+import java.util.List;
+import java.util.jar.JarEntry;
+import java.util.jar.JarFile;
+import jdk.internal.org.objectweb.asm.ClassReader;
+import jdk.internal.org.objectweb.asm.tree.AnnotationNode;
+import jdk.internal.org.objectweb.asm.tree.ClassNode;
+import jdk.internal.org.objectweb.asm.tree.MethodNode;
+
+// This class is used to compare the method annotations on the given class / jar files.
+public class AnnotationComparator {
```

Comment:
Is this class still used? Planned to be used? Or is it obsolete in favor of `AnnotationChecker`?

Reply:
Removed the class.